dom/nodes/Node-cloneNode.html

```diff
@@ -198,19 +205,19 @@
   el.setAttribute("a", "b");
   el.setAttribute("c", "d");
   var c = el.cloneNode();
-  check_copy(el, c, Element);
+  check_copy(el, c, {name: "HTMLUnknownElement"});
```

Comment:
```suggestion
  check_copy(el, c, HTMLUnknownElement);
```

would work, I think? Same for the other ones; I don't think they needed to be changed?